dev/benchmarks/macrobenchmarks/pubspec.yaml

```diff
@@ -219,4 +219,8 @@ flutter:
       fonts:
         - asset: packages/flutter_gallery_assets/fonts/GalleryIcons.ttf
 
+<<<<<<< HEAD
```

Comment:
This looks like a leftover merge artifact. It looks like it is causing several of the test failures